.gitignore

```diff
@@ -14,3 +14,4 @@ NOTES
 *.wasm
 *.ts
 *.js
+rio.log
```

Comment:
I think you can actually filter any log `*.log`

Reply:
4150cb34a6f36a6a09636a6c66a6109f496fc6ba